### PR TITLE
New functionality for create and delete socialnetwordFait divers

### PR DESCRIPTION
**Pull request by lamrani002**
# NEW|New functionality for module social networks

### CODE REVIEW

File: htdocs/admin/dict.php [hunk 1]
```diff
@@ -1261,6 +1262,10 @@
 
 llxHeader('', $title, '', '', 0, 0, '', '', '', 'mod-admin page-dict');
 
+$head = socialnetwork_prepare_head();
+
+print dol_get_fiche_head($head, 'dict', $langs->trans('MenuDict'), -1, 'user');
```
Comment (eldy) [inline htdocs/admin/dict.php:1267]: If you add a dol_get_fiche_head, you must also add a dol_get_fiche_end().

Also adding the tab must be introduce only when you open the page from the page of the module social network.
When this page is open from menu Home - dictionnary, we must keep only the simple table view with no tabs.

File: htdocs/admin/faitdivers.php [hunk 2]
```diff
@@ -0,0 +1,290 @@
+<?php
```
Comment (eldy) [inline htdocs/admin/faitdivers.php:1]: faitdiverse  must be renamed into fediverse:  https://fr.wikipedia.org/wiki/Fediverse

File: htdocs/admin/dict.php (hunk 1 above)
Comment (lamrani002) [reply htdocs/admin/dict.php:1267]: ok i see i will fix it

**Comment by eldy** [comment]
Duplicate

**Pull request closed without merging**